frontend/src/main/web/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Router from 'react-router';
+import RootContent from './lib/components/RootContent';
+import Views from './lib/constants/Views.js';
+import Configs from './lib/constants/Configs';
+import StringUtils from './lib/utils/StringUtils';
+import _ from 'lodash';
+import css from './index.css'
+
+var mountNode = document.getElementById('main-content'),
+  baseUrl = mountNode.getAttribute('base-url'),
+  user = JSON.parse(mountNode.getAttribute('user')),
+  data = JSON.parse(mountNode.getAttribute('data')),
+  view = Views.getView(data.view),
+  dev = data.dev;
```

Comment:
I'm open for any standard for formatting as long as we got a standard style for js and jsx (keep in mind Zanata coding style have formatting for js file too) and those style can be import into IntelliJ/Webstrom.